.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: groovy
+
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
```

Comment:
We don't know if the project will use JVM. Why then are we running the build against 3 versions of Java?